dags/legacy_test/tests/pytorch/nightly/common.libsonnet

```diff
@@ -141,8 +141,9 @@ local volumes = import 'templates/volumes.libsonnet';
         pip3 install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
 
         mkdir pytorch
-        wget https://github.com/pytorch/xla/archive/refs/heads/master.tar.gz -O - | tar xzf -
-        mv xla-master pytorch/xla
+        cd pytorch
+        git clone --depth=1 https://github.com/pytorch/xla.git
```

Comment:
Can you clone directly into `pytorch`? `git clone --depth=1 https://github.com/pytorch/xla.git pytorch`

Reply:
@vanbasten23 I think you'd need to `git clone https://... pytorch/xla`

Reply:
yes!